src/components/views/right_panel/GroupHeaderButtons.js

```diff
@@ -68,7 +69,7 @@ export default class GroupHeaderButtons extends HeaderButtons {
     _onMembersClicked() {
         if (this.state.phase === RIGHT_PANEL_PHASES.GroupMemberInfo) {
             // send the active phase to trigger a toggle
-            this.setPhase(RIGHT_PANEL_PHASES.GroupMemberInfo);
+            this.setPhase(RIGHT_PANEL_PHASES.GroupMemberInfo, RightPanelStore.getSharedInstance().roomPanelPhaseParams);
```

Comment:
Groups don't have phase params stored in the RightPanelStore - it's honestly probably better to just dispatch an explicit toggle request instead of trying to rely on `setPhase` doing the right thing.

Reply:
I'm not sure what a clean way to do this is without overhauling most of the RightPanelStore, we store showGroupPanel vs showRoomPanel separately, should we have a dispatch for closing each? Do you have any magic ideas here, seems like this Store is yours so maybe you know a corner we could shove it into

Reply:
yea, I don't think we can do this cleanly. Maybe it's best to just remove the extra params from this particular call, leaving the rest of the PR as it is written.